python/orca/src/bigdl/orca/learn/openvino/estimator.py

```diff
@@ -60,6 +63,7 @@ def fit(self, data, epochs, batch_size=32, feature_cols=None, label_cols=None,
     def predict(self,  # type: ignore[override]
                 data: Union["SparkXShards", "DataFrame", "np.ndarray", List["np.ndarray"]],
                 feature_cols: Optional[List[str]] = None,
+                label_cols: Optional[List[str]] = None,
```

Comment:
label_cols doesn't make sense in predict, is it a must for process_xshards_of_pandas_dataframe? If so, can we make it a fixed value?

Reply:
not a must, removed.